examples/globe_vector.html

```diff
@@ -47,17 +74,99 @@
         <script type="text/javascript">
             /* global itowns, document, GuiTools, globeView, promises */
             var menuGlobe = new GuiTools('menuDiv');
+            var tooltip = document.getElementById('tooltipDiv');
+            var mouseDown = 0;
+
             menuGlobe.view = globeView;
             // Listen for globe full initialisation event
+            tooltip.style.visibility = 'hidden';
+
+            document.body.onmousedown = function onmousedown() {
+                ++mouseDown;
+            };
+            document.body.onmouseup = function onmouseup() {
+                --mouseDown;
+            };
+
             globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function () {
                 // eslint-disable-next-line no-console
                 console.info('Globe initialized');
                 Promise.all(promises).then(function () {
                     menuGlobe.addImageryLayersGUI(globeView.getLayers(function (l) { return l.type === 'color'; }));
                     menuGlobe.addElevationLayersGUI(globeView.getLayers(function (l) { return l.type === 'elevation'; }));
                     itowns.ColorLayersOrdering.moveLayerToIndex(globeView, 'Ortho', 0);
+                    var layers = globeView.getLayers(function (l) { return l.protocol === 'rasterizer'; });
+
+                    function readPosition(e) {
+                        if (!mouseDown) {
+                            buildToolTip(globeView.controls.pickGeoPosition(e.clientX, e.clientY), e);
+                        } else {
+                            tooltip.style.left = e.pageX + 'px';
+                            tooltip.style.top = e.pageY + 'px';
+                        }
+                    }
+
+                    function pickPosition(e) {
+                        buildToolTip(globeView.controls.pickGeoPosition(e.clientX, e.clientY), e);
+                    }
+                    // precision in pixels
+                    var precisionPx = 5;
+                    function buildToolTip(geoCoord, e) {
```

Comment:
Could you move this code to a utility `examples/tooltip.js` file? It would help readability of this example and allo code sharing with other examples.